### ISSUE TITLE
Rename existing images feature create "md5_%md5%" foler

### ISSUE THREAD

**Issue by NeoZeOne** [Bionus/imgbrd-grabber#1110]
### What steps will reproduce the problem?
Rename existing images feature
Force md5 calculation
my mask (: %search_1%\%website%\%character%_%md5%.%ext%

### What is the expected behavior? What do you get instead?
It creates under %website% folder : md5_%md5%\%website%\%character%_%md5%.%ext%
I don't know if it's expected, but it would be better if images stay in their original folder

### How often does this problem occur? On which sources?
everytime

### What version of the program are you using? On what operating system?
5.5.1
Windows 10

**Comment by Bionus** [comment]
Indeed, that sounds like something it may do in some cases. Which filename format were you renaming from/to?

> It creates under %website% folder : md5_%md5%%website%%character%_%md5%.%ext%

So you mean it put the files in `%website%/md5_%md5%%website%%character%_%md5%.%ext%` instead of `%search_1%%website%%character%_%md5%.%ext%`?

**Comment by NeoZeOne** [comment]
Sorry, some characters didn't show.
The mask from/to (they are the same to rename all files on same mask) is : %character%_%md5%.%ext%
It create the path under %website% : **`md5_%md5%\%website%\%character%_%md5%.%ext%`** (variables are obviously resolved).

**Comment by Bionus** [comment]
Can I ask why you're using the same from/to files?
Wouldn't renaming the files into their original filename do nothing?

As for the issue with `%search%`, I'll see what I can do. 👌

**Comment by NeoZeOne** [comment]
I tried to explain in the previous message : they are the same to rename all files on same mask. But maybe it's not clear, I'm French, so not native English...
It's because, between two download sessions, I may have change the mask. Especially in the beginning to find the better one.
Then I wanted to rename all the files based on last mask.

I don't really understand the second question, sorry...

**Comment by Bionus** [comment]
> maybe it's not clear, I'm French, so not native English...

Feel free to speak French, I'll translate your comment afterwards.

>  It's because, between two download sessions, I may have change the mask. Especially in the beginning to find the better one.
> Then I wanted to rename all the files based on last mask.

In that case, the original filename should be your old one, and the new filename your new one.

Let's say you first downloaded 100 images with the `%md5%.%ext%` filename, but want to move them into `%artist%/%md5%.%ext%`. In that case, you should put `%md5%.%ext%` in the "origin filename" field and `%artist%/%md5%.%ext%` in the new one. You also don't need to force MD5 calculation as it's already in the filename.

If you put the same filename in both fields, Grabber will simply try to move all files that match `%artist%/%md5%.%ext%` into `%artist%/%md5%.%ext%`, so it won't rename anything.

**Comment by NeoZeOne** [comment]
Hello,
I tried to simulate what you advised me to do.
With distinct origin and destination filenames masks 
(by the way, there is a typo in this dialog box : "**Destintation**" instead of "Destination"

First, with "Get md5 in filename"
![2017-10-28 11-35-23](https://user-images.githubusercontent.com/31030470/32133280-15022c92-bbd5-11e7-8e58-a8e3560e69aa.jpg)
No images were renamed
![2017-10-28 11-36-06](https://user-images.githubusercontent.com/31030470/32133320-d1e0fabe-bbd5-11e7-9815-c85b79b6e3c7.jpg)

And then, with "Force md5 calculation"
![2017-10-28 11-37-35](https://user-images.githubusercontent.com/31030470/32133326-e97bc33e-bbd5-11e7-82ea-07b4f33c4650.jpg)

You can see that Grabber wants to rename in `md5_%md5%\%website%\%character%_%md5%.%ext%`
![2017-10-28 11-38-49](https://user-images.githubusercontent.com/31030470/32133380-e9974e32-bbd6-11e7-8e1d-bf08094cb46a.jpg)

And, when writing this post, I realized that, when omitting folder structures in masks, the rename is working perfectly!!!
My bad...
But it's not very clear what we have to put in fields.
And another thing, the txt file is not renamed as well, so it did not follow the picture name any more.

In conclusion, this is not really a bug, but a misuse of my own.
Or is it a bug after all because it creates a weird folders structure `md5_%md5%\%website%\%character%_%md5%.%ext%`?

**Comment by FichteFoll** [comment]
Your source pattern had a folder structure, but for the source folder you selected a folder with images directly. Naturally, you'll have to adjust this to your source pattern since the original pattern was based on the general download directory, not that particular one you selected.

**Comment by NeoZeOne** [comment]
Ok, thanks FichteFoll, I understand what you mean, that's now clear for me.
